package.json

```diff
@@ -21,7 +21,6 @@
   "scripts": {
     "test": "rimraf test/temp && jest --modulePathIgnorePatterns='./template'",
     "test:updateSnapshot": "rimraf test/temp && jest --updateSnapshot --modulePathIgnorePatterns='./template'",
-    "test:example": "node test/example/script.js",
```

Comment:
hmm, looks like something went wrong with solving conflicts  It is new and should stay

Reply:
also versions od some deps are downgraded